conf/scylla.yaml

```diff
@@ -245,7 +245,10 @@ batch_size_fail_threshold_in_kb: 50
 # broadcast_rpc_address: 1.2.3.4
 
```

Comment:
Is "Remove the enable_user_defined_functions option, which is now redundant." still correct? I think it's not.

Reply:
No, it's incorrect now.  Thanks for catching this.  Fixed.